src/server/PostManager.js

```diff
@@ -1,6 +1,34 @@
+import db from '../server/database';
+import {KeyUtils, ENTITY} from '../server/key-utils';
 
 export default class PostManager {
```

Comment:
■ Issue : how can we handle or deal 'type' to check whether it's item or image.
1. add type field in item and image object.
2. make createdPost json includes type field